model/DeliveryExecutionGetterInterface.php

```diff
@@ -28,9 +31,6 @@ interface DeliveryExecutionGetterInterface
 
     /**
      * Retrieves delivery execution by id (or returns null if doesn't exist) accessible by passed $ltiProvider
-     * @param string $deliveryExecutionId
-     * @param LtiProvider $ltiProvider
-     * @return DeliveryExecutionInterface|null returns null if DE doesn't exist
      */
-    public function get($deliveryExecutionId, LtiProvider $ltiProvider);
+    public function get(string $deliveryExecutionId, LtiProvider $ltiProvider): ?DeliveryExecutionInterface;
```

Comment:
There are other classes like TenantDeliveryExecutionGetter that are also implementing this interface, but method signature is different. We have to fix that, or do not change the signature.

Reply:
https://github.com/oat-sa/extension-tao-deliver-connect/pull/77